src/util/interaction/out.ts

```diff
@@ -36,6 +36,8 @@ export function progress<T>(title: string, action: Promise<T>): Promise<T> {
 // function.
 //
 export function list<T>(formatter: {(item: T): string}, items: T[]): void {
+  if (!items || Object.keys(items).length === 0) { return; }
```

Comment:
I had an issue where the response I got didn't match the responses know to the generate API client which resulted in a `{}` being returned. `Object.keys(items).length` should catch both cases, `Object` and `Array`.